library/build.gradle.kts

```diff
@@ -24,10 +24,12 @@ android {
 }
 
 dependencies {
-  api("androidx.test.espresso:espresso-core:3.1.1")
-  api("androidx.test.espresso:espresso-contrib:3.1.1")
+  api("androidx.test.ext:junit:1.0.0")
```

Comment:
I'm curious, why was this dependency added?

Reply:
I added that depency because I also updated the AndroidJUnit runner in androidx.test.runner.AndroidJUnit4 is deprecated and the new one is into androidx.test.ext.junit.runners.AndroidJUnit4. which depends on androidx.test.ext:junit.

Reply:
I think we can avoid this dependency altogether if we remove all those `@RunWith(AndroidJUnit4::class)` annotations from the tests. They are not really needed.
Then, this dependency won't be needed, right? Do you want to do it? I can send a commit otherwise.